econplayground/main/views.py

```diff
@@ -410,16 +453,16 @@ def post(self, request, *args, **kwargs):
             return HttpResponseRedirect(request.path)
 
         # Find actions that the user has made
-        rule_options = Graph.get_rule_options(self.object.graph_type)
```

Comment:
Removing the rule options here is a very drastic change to how the assessment system works. It's better to do more focused bugfixes, and it should be more clear exactly what bugs a changeset is fixing, through new unit tests.

I will keep this PR open for reference and we can possibly use parts of it, but I'm going to debug some of the assessment issues we've seen using the master branch.